data/json/itemgroups/tools.json

```diff
@@ -367,5 +367,10 @@
       [ "circsaw_blade", 10 ],
       [ "elec_chainsaw_off", 35 ]
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "slaughter_tools",
```

Comment:
This new item group doesn't seem to be used anywhere.

Reply:
Removed and slated for another PR